numpy/lib/mixins.py

```diff
@@ -5,53 +5,54 @@
 
 from numpy.core import umath as um
 
-# None of this module should be exposed in top-level NumPy module.
+# Nothing should be exposed in the top-level NumPy module.
 __all__ = []
 
 
+def _disables_array_ufunc(obj):
```

Comment:
I think it makes more sense to put this function on the class itself (and thus use `self._disables_array_ufunc` in the function definitions below). That way we define the default, but do not force people to reimplement the mixin if they do not like the default.